### ISSUE TITLE
Downloads the first file then stop, logged in Chrome with Premium account

### ISSUE THREAD

**Issue by tapishdongre** [bunnykek/kuku-dl#16]
![Bug Kuku](https://github.com/user-attachments/assets/feeb9208-a171-4ebc-a82d-55294fb51a12)

I get this error even on premium account, It downloads the first file but then stops. I have updated Python recently and then updated Mutagen and Request too. Using Windows system. Wondering is it not able to catch the account login details form Chrome ? Do we need to mannually enter these details somewehere? I remember there was a previous version (before the current update where you removed the requirement to login) How to access that file or how to download uising that version ?  Or any other solutions for the latest version ? Thanks

**Comment by bunnykek** [comment]
Yeah I need to update it.

**Comment by myselfayush010** [comment]
no, need premium account even no account copy paste link. i raised pull request.

**Comment by sathishdevi1982** [comment]
> no, need premium account even no account copy paste link. i raised pull request.

not working properly.

**Comment by myselfayush010** [comment]
true i did not checked

**Comment by sathishdevi1982** [comment]
Thanks for the reply bro.  No issues. Have you find any solution on this
issue?

**Comment by 1MrZahid** [comment]
> Yeah I need to update it.

kindly update it ... because of you i am running a channel in telegram and is serving the knowledge seekers there for free..

https://t.me/Self_Improvement_Audiobooks

thank you so much.. almighty willl reward your behavior inshallah

**Comment by sathishdevi1982** [comment]
Any update???
